streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -375,6 +377,15 @@ public boolean isStartingRunningOrPartitionAssigned() {
     private final boolean eosEnabled;
     private final boolean processingThreadsEnabled;
 
+    private final WindowedSum pollLatencyWindowedSum = new WindowedSum();
+    private final WindowedSum totalCommitLatencyWindowedSum = new WindowedSum();
+    private final WindowedSum processLatencyWindowedSum = new WindowedSum();
+    private final WindowedSum punctuateLatencyWindowedSum = new WindowedSum();
+    private final WindowedSum runOnceLatencyWindowedSum = new WindowedSum();
+    private final MetricConfig metricsConfig;
+
+    private boolean latencyWindowsInitialized = false;
```

Comment:
```suggestion
    private boolean latencyWindowsInitialized;
```
false seems to be default value for this type.

its very nice that its correctly named NOT starting with `is`, as this will be handled by java, as well.